tritonbench/utils/triton_op.py

```diff
@@ -84,13 +84,18 @@ class BenchmarkOperatorBackend:
     # need to be tested in ci
     # ci = False implies enabled = False
     ci: bool = True
+    # tags for categorizing the backend (e.g., ['triton', 'pt2'])
+    tags: Optional[List[str]] = None
 
 
 REGISTERED_BENCHMARKS: Dict[str, OrderedDict[str, BenchmarkOperatorBackend]] = {}
 REGISTERED_METRICS: defaultdict[str, List[str]] = defaultdict(list)
 OVERRIDDEN_METRICS: defaultdict[str, List[str]] = defaultdict(list)
 REGISTERED_X_VALS: Dict[str, str] = {}
 BASELINE_BENCHMARKS: Dict[str, str] = {}
+# Store tags defined in @register_benchmark decorator
+# Format: {operator_name: {backend_name: [tag1, tag2, ...]}}
+REGISTERED_BACKEND_TAGS: Dict[str, Dict[str, List[str]]] = {}
```

Comment:
Can we remove the use of `REGISTERED_BACKEND_TAGS`? Instead, we can add it as a field of `class BenchmarkOperatorBackend`